hw/system/snitch_cluster/cfg/cluster.default.hjson

```diff
@@ -91,6 +91,7 @@
     compute_core_template: {
```

Comment:
I am not sure we whether we should adapt the default cluster for this or create one or more cluster configurations for `riscv_tests`, especially if some extensions are (at least temporarily) mutually exclusive.